<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLViewerClass</class>
 <widget class="QMainWindow" name="GLViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="leFilePath">
      <property name="text">
       <string>c:\Users\fridr\source\repos\GLViewer\Data\ATAStore1D.txt </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pbLoad">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="GLWidget" name="GLWidgetMain"/>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tvCoordinates">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>File Path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="GLViewer.qrc">
     <normaloff>:/Icons/Resources/open.ico</normaloff>:/Icons/Resources/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="GLViewer.qrc">
     <normaloff>:/Icons/Resources/close.ico</normaloff>:/Icons/Resources/close.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">GLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GLViewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbLoad</sender>
   <signal>clicked()</signal>
   <receiver>actionLoad</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>909</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
